### PR TITLE
Migrate from optimist to yargs

### PR DESCRIPTION
**Pull request by Hypnosphi**
`optimist` package is [deprecated](https://github.com/substack/node-optimist#deprecation-notice) in favor of [`yargs`](https://github.com/yargs/yargs), its successor with the same [API](https://github.com/yargs/yargs/blob/master/docs/api.md)

Required for https://github.com/gemini-testing/hermione/issues/494, as `optimist` depends on a vulnerable verision of `minimist`

### CODE REVIEW

**Comment by Hypnosphi** [comment]
@miripiruni @sipayRT @xrsd can you please review

File: package.json [hunk 1]
```diff
@@ -81,7 +81,8 @@
     "supports-color": "~5.0.0",
     "url": "~0.11.0",
     "wdio-dot-reporter": "~0.0.8",
-    "wgxpath": "~1.0.0"
+    "wgxpath": "~1.0.0",
+    "yargs": "^15.4.1"
```
Comment (DudaGod) [inline package.json:85]: Why you don't remove optimist from dependencies?

**Comment by DudaGod** [comment]
Did you know that we recently released an alpha version with wdio@7? And in the near future we will deprecate our wdio fork.

**Comment by Hypnosphi** [comment]
Thanks, will check it out

**Pull request closed without merging**